admin/class-yoast-form.php

```diff
@@ -325,34 +309,29 @@ public function light_switch( $var, $label, $buttons = [], $reverse = true, $hel
 
 		list( $off_button, $on_button ) = $buttons;
 
-		$help_class = ! empty( $help ) ? ' switch-container__has-help' : '';
-
-		$strong_class = ( $strong ) ? ' switch-light-visual-label__strong' : '';
-
-		echo '<div class="switch-container', $help_class, '">',
-		'<span class="switch-light-visual-label' . $strong_class . '" id="', esc_attr( $var . '-label' ), '">', esc_html( $label ), '</span>' . $help,
-		'<label class="', $class, '"><b class="switch-yoast-seo-jaws-a11y">&nbsp;</b>',
-		'<input type="checkbox" aria-labelledby="', esc_attr( $var . '-label' ), '" id="', esc_attr( $var ), '" name="', esc_attr( $this->option_name ), '[', esc_attr( $var ), ']" value="on"', checked( $val, 'on', false ), disabled( $this->is_control_disabled( $var ), true, false ), '/>',
-		'<span aria-hidden="true">
-			<span>', esc_html( $off_button ) ,'</span>
-			<span>', esc_html( $on_button ) ,'</span>
-			<a></a>
-		 </span>
-		 </label><div class="clear"></div></div>';
+		echo '<div class="yoast-toggle__item">',
+		'<div class="yoast-toggle__item-title"><legend>', esc_html( $label ), '</legend>', $help, '</div>',
+		'<div class="', $class, '">';
+		echo '<input type="checkbox" aria-labelledby="', esc_attr( $var . '-label' ), '" id="', esc_attr( $var ), '" name="', esc_attr( $this->option_name ), '[', esc_attr( $var ), ']" value="on"', checked( $val, true, false ), disabled( $this->is_control_disabled( $var ), true, false ), '/>',
```

Comment:
This aria-labelledy points to nothing. There's no element with an ID value given by `$var . '-label'`. In the previous implementation (see above) this worked because there was a `<span>` element with ID `$var . '-label'`